app.js

```diff
@@ -4,6 +4,10 @@ console.log('loaded app.js'); //have we actually loaded app.js?
 
 function main(){
   promptUserName();
+  var wtp = wantsToPlay();
```

Comment:
My first thought when I saw the previous commit was that the `wantsToPlay` call didn't belong inside `questions()`. Glad you agree. 😀